### DISCUSSION TITLE
关于评论框的两个问题

### DISCUSSION THREAD

**Discussion by dyxang** [walinejs/waline#135] [Q&A] [answered]
1. 能否自定义无评论时评论框下方的“来发评论吧~”？
2. 能否仔细选择是否显示“登录”按钮？因为大多时候“登录”给人一种注册才能评论的感觉，然后就是手机屏幕较小时登录独占左侧区域，显着旁边的评论框太窄

**Comment by lizheming** [comment, marked as answer]
1. 文案自定义可以通过自定义 i18n 配置来实现
    ```js
    new Waline({
      lang: 'zh-CN',
      langMode: {
        sofa: '你有权保持沉默。如果你开口说话，那么你所说的每一句话都将作为呈堂证供。'
      }
    });
    ```
2. 最新的版本里已经支持了 `anonymous` 参数，用于控制登录的显示，设置 `anonymous: true` 即可隐藏登录按钮，具体见文档 https://waline.js.org/client/basic.html#anonymous。

**Comment by redstoneleo** [comment]
Waline隐藏登陆按钮 目前应该看 https://waline.js.org/reference/client/props.html#login
